linera-views/src/scylla_db.rs

```diff
@@ -373,8 +372,10 @@ impl ReadableKeyValueStore<ScyllaDbContextError> for ScyllaDbStoreInternal {
         let store = self.store.deref();
         let _guard = self.acquire().await;
         let handles = keys.into_iter().map(|key| store.read_value_internal(key));
-        let result = join_all(handles).await;
-        Ok(result.into_iter().collect::<Result<_, _>>()?)
+        join_all(handles)
+            .await
+            .into_iter()
+            .collect::<Result<_, _>>()
```

Comment:
Nice.

Reply:
This (and other small rewrites in this PR) were mandated by Clippy after removing `async-trait`!